app/vader_sentiment.py

```diff
@@ -1,7 +1,6 @@
 from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
 vader = SentimentIntensityAnalyzer()
 
```

Comment:
The blank line that was deleted here should be there. This is part of PEP8, all functions should be separated by 2 blank lines.